ui/src/alerting/components/CheckEOHeader.tsx

```diff
@@ -98,10 +101,7 @@ const CheckEOHeader: FC<Props> = ({
   }
 
   const {singleField, singleAggregateFunc} = isDraftQueryAlertable(draftQueries)
-  const oneOrMoreThresholds =
-    check.type === 'threshold'
-      ? check.thresholds && !!check.thresholds.length
-      : false
+  const oneOrMoreThresholds = checkType === 'threshold' && !!thresholds.length
```

Comment:
I guess I was being a little more lax about this since thresholds is coming from redux, and not as a parameter from check, but we do overwrite alertBuilder.thresholds from a check coming from the API. so will add extra safety here